cluster/saltbase/salt/kube-apiserver/default

```diff
@@ -27,6 +27,10 @@
 {% if grains.cloud == 'gce' -%}
   {% set cloud_provider = "--cloud_provider=gce" -%}
 {% endif -%}
+{% if grains.cloud == 'aws' -%}
+  {% set cloud_provider = "--cloud_provider=aws" -%}
+  {% set cloud_config = "--cloud_config=/etc/aws.conf" -%}
```

Comment:
this broke Vagrant.

In the future, please always set the initial value of a global variable to the empty string outside of an if-block.

I think this would have also broke other providers.

Will send a patch.